src/GitHub.VisualStudio/Styles/ThemeDesignTime.xaml

```diff
@@ -15,9 +15,30 @@
   <SolidColorBrush x:Key="GitHubVsCommandBarSelectedBorder" Color="#FF3399FF" />
   <SolidColorBrush x:Key="GitHubVsSearchBoxBackground" Color="#FF333337" />
   <SolidColorBrush x:Key="GitHubVsWindowText" Color="#FFF1F1F1" />
+  <SolidColorBrush x:Key="GitHubVsTextBoxBorder" Color="#FF3F3F46" />
+  <SolidColorBrush x:Key="GitHubVsTextBoxBackground" Color="#FF2D2D30" />
+
+  <SolidColorBrush x:Key="GitHubVsButtonBackgroundBrush" Color="#FF3F404B" />
```

Comment:
You don't need to add colors to `ThemeDesignTime.xaml` if they're already defined in one of the ThemeXX.xaml files. The only things you need to add to the `ThemeDesignTime.xaml` file is colors that are defined in `VsBrush.xaml`, because that file only has dynamic resources that aren't available at design time.

TL;DR: Kill all these brushes except for `GitHubVsTextBoxBorder` and `GitHubVsTextBoxBorder`